README.md

```diff
@@ -4,6 +4,24 @@
 [gitter-image]: https://badges.gitter.im/open-telemetry/opentelemetry-python.svg
 [gitter-url]: https://gitter.im/open-telemetry/opentelemetry-python?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
 
+We hold regular meetings. See details at [community page](https://github.com/open-telemetry/community#python-sdk).
+
+Approvers ([@open-telemetry/python-approvers](https://github.com/orgs/open-telemetry/teams/python-approvers)):
```

Comment:
Allan is also an approver currently isn't he?
https://github.com/a-feld

Reply:
I do recall him being added. I think the teams simply need to be updated.
@c24t @a-feld  Correct?

Reply:
He's not in approvers, but he is in [CODEOWNERS](https://github.com/open-telemetry/opentelemetry-python/blob/9290eaf27d9ba1b426a3358c8297c1c03ca913ab/.github/CODEOWNERS#L5).

Following the [org approver/maintainer guidelines](https://github.com/open-telemetry/community/blob/0b0ff608c6870e2d5ef2fd45dcb3120e3fd3a622/community-membership.md#approver), approvers have to meet time and PR count requirements. We may want to keep the approvers list and CODEOWNERS file in sync, but for now I think it's fine for CODEOWNERS to include anyone actively working on the project. It doesn't grant any permissions, just notifies users on new issues and PRs.

Reply:
Actually we have the same nomination process for CODEOWNERS, i.e. members: https://github.com/open-telemetry/community/blob/e148326db4b38a122ac1aeb7da76719816dcf558/community-membership.md#member.